apps/academy/babel.config.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
```

Comment:
are unifying the eslint configs and applying auto fixes, there are many changes like this, where things like single quotes are swapped for double etc. we need to do this because otherwise we cannot just run eslint at the top level, since there will be inconsistent results.